CommunityFridgeMapApi/dependencies/python/db.py

```diff
@@ -191,48 +191,39 @@ def add_item(self) -> DB_Response:
     def get_fridge_locations(self):
         pass
 
-
 class FrigeCheckIn(DB_Item):
-
     def __init__(self, db_client: 'botocore.client.DynamoDB'):
         super().__init__(db_client=db_client)
 
 class FridgeHistory(DB_Item):
-
     def __init__(self, db_client: 'botocore.client.DynamoDB'):
         super().__init__(db_client=db_client)
 
 
 class Tag(DB_Item):
     REQUIRED_FIELDS = ['tag_name']
     TABLE_NAME = "tag"
+
     def __init__(self, db_client: 'botocore.client.DynamoDB', tag_name:str= None):
         super().__init__(db_client=db_client)
-        self.tag_name = tag_name
+        self.tag_name = self.format_tag(tag_name)
 
     def format_tag(self, tag_name:str) -> str:
         #Tag_name is alphanumeric, can include hyphen and underscore, with no spaces and all lower cased
         if tag_name:
             tag_name = tag_name.lower().replace(" ", "")
-            if not self.is_valid_tag_name(tag_name):
-                return False
-            else:
-                self.tag_name = tag_name
-                return True
-        else:
-            return False
+        return tag_name
 
     def is_valid_tag_name(self, tag_name:str) -> bool:
-        #A valid tag name is alphanumeric, can include hyphen, underscore
-        if tag_name:
+        #A valid tag name is alphanumeric, can include hyphen, underscore, with no spaces and all lower cased
+        if tag_name and len(tag_name) >= 3:
```

Comment:
nit: min tag length should be defined as a constant imo

Reply:
I agree